models/sage_intacct__ap_ar_enhanced.sql

```diff
@@ -80,7 +80,7 @@ ap_bill_enhanced as (
         cast(null as {{ dbt.type_string() }}) as bill_item_id,
         ar_invoice_item.invoice_id,
         ar_invoice_item.invoice_item_id,
-        ar_invoice_item.account_no,
+        cast(ar_invoice_item.account_no as {{ dbt.type_string() }}) as account_no,
```

Comment:
That's a great callout! I wasn't able to find any `offset_gl_account_no` fields that had the same stringy results as the normal `account_no`. However, that should not be an exception here (especially since we don't cast it upstream). I agree with you that it would be safest to cast these just in case, even if they are not a problem now. They may be in the future.

Reply:
Updated